docs/understanding-tracking-design/organize-data-sources-with-source-applications/index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Organize Data Sources with Source Applications"
+sidebar_position: 1
+sidebar_label: "🆕 Source Applications"
+---
+
+For data collection, you will often have different sources of information that correspond to applications designed for a particular purpose. That is what we will be referring to as Source Applications.
+
+To illustrate an example, for Snowplow we can discern more than a couple of applications designed for a specific purpose and each is a separate data source for behavioral data, or better yet a Source Application:
+- The Snowplow website that corresponds to the application served under www.snowplow.io
+- The BDP Console application that is served under console.snowplowanalytics.com.
+- The documentation website serving as our information hub, for all things related to our product, served under docs.snowplow.io.
+
+Source Applications are a foundational component that will allow you to create the overarching relationship connecting application IDs, [Application Entites](../../collecting-data/collecting-from-own-applications/javascript-trackers/web-tracker/custom-tracking-using-schemas/global-context/index.md) and [Data Products](../defining-the-data-to-collect-with-data-poducts/index.md).
+
+## Application IDs
+
+For each of these applications you would set up a unique application ID using the [app_id](../../collecting-data/collecting-from-own-applications/snowplow-tracker-protocol/ootb-data/app-information/index.md#atomic-event-properties) field to discern them later on in analysis.
+
+:::tip
+What we often see and consider a best practice is to also set up an application ID for each deployment environment you are utilizing. For example `${appId}-qa` for staging, `${appId}-dev` for development environments.
```

Comment:
We often see, and recommend as a best practice, setting up a unique application ID for each deployment environment you are using.